apps/01PigLatin.js

```diff
@@ -4,13 +4,53 @@ var assert = require('assert');
 var prompt = require('prompt');
 prompt.start();
 
-
 function pigLatin(word) {
 
-    // Your code here
+  //Attaches "yay" if word begins with vowel:
+  if(word[0] === 'a' || word[0] === 'e' || word[0] === 'i' || word[0] === 'o' || word[0] === 'u' ) {
+      return word + "yay";
+  }
+  //Translates a simple word:
+  var firstLetter = word[0];
+  return word.replace(firstLetter, '') + firstLetter + 'ay';
+  //Auto lowercases word before translation
+  word.toLowerCase()
+  //Translates a complex word:
```

Comment:
@dorianpenaloza `word.toLowerCase()` returns a new word. Word is unchanged by this operation.

`word = word.toLowerCase()` would do the job. Does this make sense?

Also, this should be the first line of code in your function, before you do any processing of the word.